ui/src/views/infra/network/TrafficTypesTab.vue

```diff
@@ -129,6 +140,35 @@ export default {
       }).finally(() => {
         this.fetchLoading = false
       })
+
+      this.fetchLoading = true
+      api('listZones', { id: this.resource.zoneid }).then(json => {
+        const zone = json.listzonesresponse.zone || []
+        this.networkType = zone[0].networktype
+      }).catch(error => {
+        this.$notifyError(error)
+      }).finally(() => {
+        this.fetchLoading = false
+      })
+    },
+    fetchGuestNetwork () {
+      if (this.networkType === 'Basic') {
```

Comment:
@Pearl1594 Can you please check if this even called. Default value for `networkType` is `Advanced` and `fetchGuestNetwork` is called before finding `networkType` at line 147. I think `network` object of IpRangesTabPublic.vue will always remain empty.